src/test/java/org/jboss/forge/arquillian/util/DependencyUtilTest.java

```diff
@@ -77,7 +77,7 @@ public void should_get_all_available_versions_if_not_supported_by_chameleon() th
         deps.add(DependencyBuilder.create().setVersion("7.1").getCoordinate());
 
         List<String> dep = DependencyUtil.toVersionString(deps,
-            createContainer("arquillian-jbossas-managed-4.2", "JBoss As"));
+            createContainer("org.jboss.as", "arquillian-jbossas-managed-4.2", "JBoss As"));
 
         assertThat(dep).contains("4.1", "7.1");
```

Comment:
These tests are very strange. We are first creating `deps` which only have versions. Then we transform it to `dep` (confusing name btw) to JBoss AS 4.2 container from which we extract versions and we assume it will consist of "4.1" and "7.1". Is there `org.jboss.as:arquillian-jbossas-managed-4.2:7.1` available? Because that's how I understand those tests.

This also shows weakness in `DependencyUtil.toVersionString` as we only check version match - that's why it all works against unrealistic dependencies.